lib/generators/react_on_rails/generator_helper.rb

```diff
@@ -17,6 +17,16 @@ def puts_setup_file_error(file, data)
     puts "\n#{data}\n"
   end
 
+  def return_setup_file_error(file, data)
+    data = <<-DATA.strip_heredoc
```

Comment:
@justin808 @aaronvb This code is a little confusing to me because you're assigning the parameter variable `data` to a variable called `data` that is set to a string that interpolates `data`, and then you add a call to `data` at the end, but I think this returns the same thing that would have been returned had you left it out. 

Anyway, the reason this indentation is happening is because `strip_heredoc` only strips to the leftmost non-space character found out of all the lines, and since you are including `#{data}` which has lines in it that are completely left-aligned, then strip-heredoc can't really do anything.

Example: 

```
 <<-TEXT.strip_heredoc
                  some text
                    some more text
TEXT 

# ouputs:
some text
   some more text
```

```
 <<-TEXT.strip_heredoc
                  some more text
                    some more text also
some text
TEXT 

# outputs:
                  some more text
                    some more text also
some text
```